docs/content/stepbystep/Git-setup.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Git setup'
+metaTitle: 'Installing and using git | Discordeno'
+metaDescription: 'Setting up git for your pc'
+---
+
+# Installing
+
+First, go to the [Git download page](https://git-scm.com/downloads) and download the version for your OS. I'm using windows so I'll use <https://git-scm.com/download/win>
+
+Once downloaded click next and next again(unless you need to change the install location)
+
+Make sure that on the select component screen all these are selected:
+
+- Git LFS
+- Associate .git files
+
+I have some extras selected as seen below. These are optional but will help
```

Comment:
`I` -> `we`

Reply:
those are things i personally enable, they dont need them

Reply:
@Suyashtnt, in that case, you might as well remove it.